### DISCUSSION TITLE
Force thumbnail generation option under extra networks

### DISCUSSION THREAD

**Discussion by i-liek-katz** [vladmandic/sdnext#1689] [Ideas]
Currently, it *seems* a thumbnail for an extra network file will generate when a dimension is larger than x, unsure what x is. I've had files much smaller in filesize generate a thumbnail, while some larger files be left alone. Maybe an option could be added to define what kind of parameters must be met to generate a thumbnail, or perhaps an option to always generate a thumbnail with a dimension of 512px by whatever.

**Comment by vladmandic** [comment]
rule for creating thumbnails is: `if img.width > 1024 or img.height > 1024:`
